networkx/algorithms/tournament.py

```diff
@@ -148,8 +148,9 @@ def random_tournament(n, seed=None):
 
     Returns
     -------
-    bool
-        Whether the given graph is a tournament graph.
+    G : DiGraph
+        A tournament on n nodes, with exactly one directed edge joining
```

Comment:
```suggestion
        A tournament on `n` nodes, with exactly one directed edge joining
```
Per the numpydoc standard